app/controllers/users/relationships_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Users::RelationshipsController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+    following = current_user.active_relationships.build(followed_id: params[:followed_id])
+    following.save
```

Comment:
save の戻り値を見ないのであればこれでいいですね。
```suggestion
    current_user.active_relationships.create!(followed_id: params[:followed_id])
```